packages/components/modules/messages/web/GroupChatCreate/Header/index.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+
+import { FC } from 'react'
+
+import { IconButton } from '@baseapp-frontend/design-system/components/web/buttons'
+import { CheckMarkIcon, CloseIcon } from '@baseapp-frontend/design-system/components/web/icons'
+
+import { Typography } from '@mui/material'
+
+import { HeaderContainer } from './styled'
+import { HeaderProps } from './types'
+
+const Header: FC<HeaderProps> = ({ isDisabled, onCreateButtonClicked, onBackButtonClicked }) => (
```

Comment:
I don't know if this is the idea here but maybe we could allow the change of the close icon and Checkmark icon via props.
And keep this 2 as the Default here